sdk/spring/CHANGELOG.md

```diff
@@ -281,6 +396,12 @@ This section includes changes in `spring-cloud-azure-core`, `spring-cloud-azure-
 
 Please refer to [Spring Cloud Azure Migration Guide for 4.0][Spring-Cloud-Azure-Migration-Guide-for-4.0] to learn how to migrate to version 4.0.
 
+### Spring Cloud Azure Autoconfigure
+This section includes changes in `spring-cloud-azure-autoconfigure` module.
+
+#### Breaking Changes
+- Remove the health indicator for `KeyVaultEnvironmentPostProcessor` [https://github.com/Azure/azure-sdk-for-java/pull/24309](https://github.com/Azure/azure-sdk-for-java/pull/24309).
```

Comment:
how about keeping the same link format

`[#24309](https://github.com/Azure/azure-sdk-for-java/pull/24309)`